test_cromwell.sh

```diff
@@ -11,51 +11,50 @@ trap "shutdown" EXIT
 
 set -e
 
-EXIT_CODE=31
+EXIT_CODE=1
 PROGNAME="$(basename $0)"
 
 usage="
-$PROGNAME [-b branch] [-r rundir] [-c config file] [-p parallelism factor]
+$PROGNAME [-b branch] [-j jar path] [-r rundir] [-c config file] [-p parallelism factor]
 
 Builds and runs specified branch of Cromwell and runs Centaur against it.
 
 Arguments:
-    -b    Branch of Cromwell to test
+    -b    Branch of Cromwell to test. Mutually exclusive with -j
+    -j    Path of a cromwell jar to use. Mutually exclusive with -b
     -r    Directory where script is run (defaults to current directory)
     -c    If supplied, the config file to pass to Cromwell
     -p    Number of simultaneous tests to run. Defaults to 3
 "
 
 INITIAL_DIR=$(pwd)
-
-PARALLELISM_FACTOR=3
-CROMWELL_BRANCH="develop"
 RUN_DIR=$(pwd)
-CONFIG_STRING=""
 
-while getopts ":hb:r:c:p:" option; do
+while getopts ":hb:r:c:p:j:" option; do
     case "$option" in
-	h) echo "$usage"
-	   exit
-	   ;;
-	b) CROMWELL_BRANCH=$OPTARG
-	   ;;
+        h) echo "$usage"
+            exit
+            ;;
+        b) CROMWELL_BRANCH=$OPTARG
+            ;;
         r) RUN_DIR=$OPTARG
-	   mkdir -p "${RUN_DIR}"
-	   ;;
+            mkdir -p "${RUN_DIR}"
+            ;;
         c) CONFIG_STRING="-Dconfig.file=$OPTARG"
```

Comment:
What I meant was, you want the `"`s to appear in the result string, so that `CONFIG_STRING` would end up looking like `-Dconfig.file="hello world.txt"` instead of `-Dconfig.file=hello world.txt`